version.py

```diff
@@ -14,7 +14,7 @@
 GIT_REF_LEN = 12
 
 
-def exec(command, cwd=None):
+def exec(command, cwd=os.path.dirname(sys.argv[0])):
```

Comment:
Can you instead remove the `=None` here and this to the `exec` call on line 63?

Can you also remove the `='.'` from `git_commit`?  All the callers the directory name explicitly.

Reply:
Done